CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 2.12.0 ##
 * Fixed error message while get token from metadata with asyncio iam
 * Add `SnapshotReadOnly` transaction mode support to `session.transaction`
+* Support `TYPE_UNSPECIFIED` item type to scheme ls
```

Comment:
changes must be written to top of changelog.

version 2.12.0 is published already and doesn't include TYPE_UNSPECIFIED.

Reply:
right, I didn't notice